code/modules/cargo/expressconsole.dm

```diff
@@ -0,0 +1,110 @@
+/obj/machinery/computer/cargo/express
+	name = "express supply console"
+	desc = "This console allows the user to purchase a package for double the price,\
+		with 1/40th of the delivery time: made possible by NanoTrasen's new \"Drop Pod Railgun\".\
+		All sales are near instantaneous - please choose carefully"
+	icon_screen = "supply_express"
+	circuit = /obj/item/circuitboard/computer/cargo/express
+	blockade_warning = "Bluespace instability detected. Delivery impossible."
+	var/message
+	var/locked = TRUE
+	req_access = list(ACCESS_QM)
```

Comment:
move req_access above the var/ declarations

Reply:
:chad: